modules/ROOT/pages/testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc

```diff
@@ -149,4 +149,4 @@ If you see a similar output, this means your test is executed successfully.
 * link:https://quarkus.io/guides/getting-started-testing[Testing a Quarkus application]
 * link:https://rest-assured.io/#docs[REST Assured Documentation]
 
-include::../../pages/_common-content/report-issue.adoc[]
+include::../../pages/_common-content/report-issue.adoc[]
```

Comment:
Do we want to include this part in openshift docs ?
Or maybe should we change the link inside to it points to Red Hat docs ?

Reply:
We can change the link based on the downstream/upstream context.